robotpos1:
  controller:
    state:
      type: joint_state_controller/JointStateController
      publish_rate: 50
    position:
      joint_1:
        type: position_controllers/JointPositionController
        joint: joint_1
        pid: {p: 10.0, i: 0.0, d: 1.0}